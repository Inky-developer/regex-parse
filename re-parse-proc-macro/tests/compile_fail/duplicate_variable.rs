use re_parse_proc_macro::re_parse;

fn main() {
    let var: u32;
    re_parse!("{var}B{var}", "111B222");
    let _ = var;
}