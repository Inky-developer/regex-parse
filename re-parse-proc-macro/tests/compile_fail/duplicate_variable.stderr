error: proc macro panicked
 --> tests/compile_fail/duplicate_variable.rs:5:5
  |
5 |     re_parse!("{var}B{var}", "111B222");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: (TODO MAKE ERROR) Variable "var" is already declared
